examples/mocking/lib.rs

```diff
@@ -61,7 +61,7 @@ mod tests {
             ContractMock::new().with_message(CALLEE_SELECTOR, mock_message(|()| RETURN_VALUE));
 
         // Secondly, we deploy it, similarly to a standard deployment action.
-        let mock_address = session.mocking_api().deploy(mocked_contract);
+        let mock_address = session.sandbox().deploy(mocked_contract);
```

Comment:
I think it's a regression that users now have to even know about the `Sandbox`. It was supposed to be a low-level object and developers use high-level `session`.

Reply:
yeah right this is indeed a bit confusing without the "mocking_api" name
will update

Reply:
It's not only about the name - name is just a "gate" into the `Sandbox` world. Of course, we can't stop users from using it, if they want, but I think the API should lead developer through the path of least resistance and suggest usage of simple constructs.

Reply:
I think the issue here is that the mock utilities should not be tied to the Sandbox struct.
I restored the API and moved these props to the session in the last commit.
lmk what you think

Reply:
Looks good, thanks :)